src/Git/Git.cpp

```diff
@@ -2149,6 +2149,27 @@ CString CGit::GetGitSystemConfig() const
 
 BOOL CGit::CheckCleanWorkTree(bool stagedOk /* false */)
 {
+	if (m_IsUseLibGit2)
+	{
+		git_status_options statusopt = GIT_STATUS_OPTIONS_INIT;
+		git_status_list *status;
+		BOOL isClean = FALSE;
+		int errCode;
+
+		statusopt.show = GIT_STATUS_SHOW_WORKDIR_ONLY;
+		statusopt.flags = GIT_STATUS_OPT_UPDATE_INDEX | GIT_STATUS_OPT_EXCLUDE_SUBMODULES;
+
+		errCode = git_status_list_new(&status, g_Git.GetGitRepository(), &statusopt);
+
+		if (errCode == 0 && status)

```

Comment:
Initialize it is a good idea. :)
BTW, according to the source code, ```status``` is initialized inner ```git_status_list_new()``` with ```NULL``` at the beginning, but the behavior of ```git_status_list_new``` might be changed in the future.

According to the doc, need to check the return code.